webpack.config.js

```diff
@@ -123,6 +123,7 @@ const webpackConfig = {
 			},
 			{
 				test: /\.s?css$/,
+				exclude: /storybook\/wordpress/,
```

Comment:
We don't want this plugin working on the wordpress CSS files imported for storybook.